3] guessing game 2/main.py

```diff
@@ -0,0 +1,36 @@
+def computer_guess():
```

Comment:
celkem zajimave reseni ktere asi funguje. Jen je pro mne trochu matouci jak to guess inicialiuzujes dvakrat, jednou pred tim while  a pak uvnitr.

pak taky ta logika logika pricitani a odcitani tech attemps kdyz ho prictes ale kdyz to byl spatny feedback tak ho hned zas odectes v te else vetvi - trochu komplikovane ale celkove zatim asi jedno z nejlepsich reseni

Good job